spot-electron/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": [
+        "eslint-config-jitsi",
+        "eslint-config-jitsi/jsdoc"
+    ],
+    "rules": {
+        "no-console": 1,
+        "space-before-function-paren": ["error", {
+            "anonymous": "always",
+            "named": "never",
+            "asyncArrow": "always"
+        }]
+    },
```

Comment:
Why the rule override for space-before-function-paren? I wonder why I did the same for spot-webdriver...

Reply:
I reused your file :D